tests/core/tests/test_widgets.py

```diff
@@ -264,7 +264,7 @@ def test_clean(self):
         self.assertEqual(self.widget.clean(11.111), self.value)
 
     def test_render(self):
-        self.assertEqual(self.widget.render(self.value), self.value)
+        self.assertEqual(self.widget.render(self.value), "11.111")
```

Comment:
After merging the tests, one error appear in `test_render()` because `'11.111' != self.value`. I've changed the test itself because `render()` should return a representation of the value as stated in the `Widget` declaration.